CHANGES/1076.breaking.rst

```diff
@@ -0,0 +1,3 @@
+Removed support :rfc:`3986#section-3.2.3` port normalization when the scheme is not one of ``http``, ``https``, ``wss``, or ``ws`` -- by :user:`bdraco`.
+
+Support for port normalization was recently added in :issue:`1033` and contained code that would do blocking I/O if the scheme was not one of the four listed above. The code has been removed because this library is intended to be safe for usage with ``asyncio``.
```

Comment:
FYI
```rst
:mod:`asyncio`
```
is a thing. I made https://webknjaz.github.io/intersphinx-untangled/docs.python.org/ to explore the available objects in-browser, in case that helps.

Reply:
Thanks, missed it by a couple seconds, but I’ll do a post release merge to clean it up